.env.development

```diff
@@ -24,3 +26,5 @@ TID_SIGN_IN_CLIENT_ID=claim
 
 ONELOGIN_SIGN_IN_ISSUER=https://oidc.integration.account.gov.uk/
 ONELOGIN_REDIRECT_BASE_URL=http://localhost:3000
+
+EY_MAGIC_LINK_SECRET=some-secret-for-ey-magic-link
```

Comment:
given our current `SECRET` is a constant that is random does this mean every time the app reboots OTPs used elsewhere are no longer valid?